middlewares/ratingValidation.js

```diff
@@ -0,0 +1,44 @@
+import { isNumeric } from 'validator';
+
+export default {
+  verifyMentor(req, res, next) {
+    const { isMentor } = req.decoded;
+    if (isMentor === false) {
+      const error = new Error('Only mentors can rate articles');
+      error.status = 401;
+      return next(error);
+    }
+    return next();
+  },
+
+  validateInput(req, res, next) {
```

Comment:
The variable name here could better capture what it is validating. For example, `isRateValueSupplied`